index.html

```diff
@@ -0,0 +1,130 @@
+            href="https://fonts.googleapis.com/css2?family=Inter:wght@500"
+        />
+        <link
+            rel="stylesheet"
+            href="https://fonts.googleapis.com/css2?family=Poppins:wght@700"
+        />
+        <link
+            rel="stylesheet"
+            href="https://api.fontshare.com/css?f[]=poppins@700&display=swap"
+        />
+        <title>HTML Preview</title>
+        <link rel="stylesheet" type="text/css" href="styles.css" />
+    </head>
+    <body>
+        <div class="toolbar-dark-compact-box">
+            <div class="toolbar-dark-compact-logo-placeholder">
+                <p class="toolbar-dark-compact-my-logo">My Logo</p>
+                </div>
+                <div class="toolbar-dark-compact-normal-button-tertiary-enabled">
+                    <div class="toolbar-dark-compact-icons-icon-menu">
+                        <svg
+                            width="100px"
+                            height="100%"
+                            preserve-aspect-ratio="none"
+                            view-box="0 0 24 24"
+                            fill="none"
+                            xmlns="http://www.w3.org/2000/svg"
+                        >
+                            <path
+                                fill-rule="evenodd"
+                                clip-rule="evenodd"
+                                d="M 3 7 C 3 6.448 3.448 6 4 6 H 20 C 20.552 6 21 6.448 21 7 C 21 7.552 20.552 8 20 8 H 4 C 3.448 8 3 7.552 3 7 Z M 3 12 C 3 11.448 3.448 11 4 11 L 20 11 C 20.552 11 21 11.448 21 12 C 21 12.552 20.552 13 20 13 L 4 13 C 3.448 13 3 12.552 3 12 Z M 4 16 C 3.448 16 3 16.448 3 17 C 3 17.552 3.448 18 4 18 H 20 C 20.552 18 21 17.552 21 17 C 21 16.448 20.552 16 20 16 H 4 Z"
+                                fill="#6070FF"
+                            ></path>
+                        </svg>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </body>
+</html>
+   <head>
+            <link
+            rel="stylesheet"
+            href="https://fonts.googleapis.com/css2?family=Inter:wght@500"
+        />
+            <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700" /><link rel="stylesheet" href="https://api.fontshare.com/css?f[]=poppins@400,500,700&display=swap" />
+        <title></title>
+        <link rel="stylesheet" type="text/css" href="styles.css" />
+    </head>
+    <body>
+        <div> <div
+        class="headline-headline"
+    ><p
+    <img src="header-background-mobile.png"
+    <img src="header-shapes mobile.png"
+    class="headline-primary-text"
+><br>
+I’m Wachira Nderitu Glad to see you!
+</br>
+</p>
+<p
+    
+    class="headline-primary-text-1"
+>I’m a software developer! I reside in Nairobi, Kenya. I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+<div
```

Comment:
Kindly wrap your headline in a `<section>` tag instead of a `<div>` tag🙏. This is for semantic reasons as section tags are used to represent a generic standalone section of the document